sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -459,6 +459,38 @@ def test_records_traverse_transform_with_mocks(self):
 
       assert_that(jobs, equal_to([job_reference]), label='CheckJobs')
 
+  def test_load_job_id_used(self):
```

Comment:
Can you also manually run a pipeline to confirm that this works end to end ? (no need to add an integration test).

Reply:
I've done this, and it works e2e